subprojects/docs/src/docs/release/notes.md

```diff
@@ -97,7 +97,7 @@ Note that JUnit 5 requires Java 8 or higher.
 
 Gradle now creates a dependency constraint for each dependency declaration in a POM file with `<optional>true</optional>`. This constraint will produce the expected result for an optional dependency: if the dependency module is brought in by another, non-optional dependency declaration, then the constraint will apply when choosing the version for that dependency (e.g., if the optional dependency defines a higher version, that one is chosen).
 
-_Note:_ This is a _Gradle 5.0 feature preview_, which means it is a potentially breaking change that will be activated by default in Gradle 5.0. It can be turned on in Gradle 4.6+ by setting `org.gradle.advancedpomsupport=true` in _gradle.properties_.
+_Note:_ This is a _Gradle 5.0 feature preview_, which means it is a potentially breaking change that will be activated by default in Gradle 5.0. It can be turned on in Gradle 4.6+ by adding `enableFeaturePreview('IMPROVED_POM_SUPPORT)` in _settings.gradle_.
```

Comment:
:x: `enableFeaturePreview('IMPROVED_POM_SUPPORT)` is missing a - **`'`** should be `'IMPROVED_POM_SUPPORT'`

In all the places :)

Reply:
That's the power of copy paste for me 😉 Fixing